DictionaryCompressor/FileCompressor.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+
+namespace DictionaryCompressor
+{
+    internal class FileCompressor : IFileCompressor
+    {
+        public void CompressFile(string filePath, string output)
```

Comment:
File handling and compression are two concerns, so this class violates the Single Responsibility Principle. Split off a class that compresses a `Stream`.

Reply:
Addressed in: fb72015